src/WebJobs.Script/Config/ScriptHostConfiguration.cs

```diff
@@ -10,12 +10,15 @@ namespace Microsoft.Azure.WebJobs.Script
 {
     public class ScriptHostConfiguration
     {
+        public const int DefaultMaximumFunctionCount = 100;
+
         public ScriptHostConfiguration()
         {
             HostConfig = new JobHostConfiguration();
             FileWatchingEnabled = true;
             RootScriptPath = Environment.CurrentDirectory;
-            RootLogPath = Path.Combine(Path.GetTempPath(), "Functions");            
+            RootLogPath = Path.Combine(Path.GetTempPath(), "Functions");
+            MaximumFunctionCount = DefaultMaximumFunctionCount;
```

Comment:
nit: MaxFunctionCount. Matches the core SDK naming scheme (e.g. config.Queues.MaxDequeueCount)